javascript/code-challenges/repeatedWord/README.md

```diff
@@ -0,0 +1,16 @@
+# Repeated Word
+
+Resolved by Mariko Alvarado, Simon Panek and Ricardo Barcenas, Nathan Rhead
+
+## Challenge
+Write a function that accepts a string and returns the first repeating word.
+
+## Approach & Efficiency
+
+
+
+## Solution
+
+- [Repeated Word](repeated-word.js) 
+
+![Whiteboard](./assets/repeated-word-cc31.png)
```

Comment:
Looks like you are off to a great start with your whiteboard, when you have more time to come back to this I will regrade for you!